src/vmm/src/version_map.rs

```diff
@@ -41,6 +45,7 @@ lazy_static! {
 
         mapping.insert(String::from("0.24.0"), 2);
         mapping.insert(String::from("0.25.0"), 3);
+        mapping.insert(String::from("0.26.0"), 4);
```

Comment:
I see here the full mapping. Since we're using also constants (`FC_V0_23_SNAP_VERSION`), we should centralize their definitions and also use them here.

Reply:
Nice. Fixed.